bot/bot.go

```diff
@@ -10,6 +10,7 @@ import (
 	"strings"
 
 	"github.com/bobheadxi/calories/facebook"
+	"github.com/bobheadxi/calories/scheduler"
```

Comment:
Also you don't have a `package` named `scheduler` anywhere, I dont think this will compile haha

Before submitting make sure you `go build` in the command line